static/js/notifications.js

```diff
@@ -228,11 +228,16 @@ function flash_pms() {
 }
 
 exports.update_pm_count = function () {
-    // TODO: Add a `window.electron_bridge.updatePMCount(new_pm_count);` call?
     if (!flashing) {
         flashing = true;
         flash_pms();
     }
+    if (window.electron_bridge !== undefined) {
+        window.electron_bridge.send_event('unread_pm_count', {
+            unread_pm_count: unread.get_counts().private_message_count,
+            realm_uri: page_params.realm_uri,
```

Comment:
@akashnimare we shouldn't need a `realm_uri` parameter in requests to the `electron_bridge`, right, since the electron app knows which realm it is anyway?

Reply:
@timabbott I think we need this parameter because we need to know which tab to update in the sidebar. Currently, we read that from the page title whenever that updates, but this is meant to specifically notify the user that a PM has arrived. Since the user may have enabled notifications for all messages but still wants an extra nudge for PMs, we need to add message counts based on title updates and get the PM count based on the realm URI.